cloudapi/logs.go

```diff
@@ -100,6 +100,8 @@ func (c *Config) logtailConn(ctx context.Context, referenceID string, since time
 
 	headers := make(http.Header)
 	headers.Add("Sec-WebSocket-Protocol", "token="+c.Token.String)
+	headers.Add("Authorization", "token "+c.Token.String)
```

Comment:
This has the same token as the above? 

Can't we just keep the old one?

Reply:
Does that mean that this will stop working with just `Sec-WebSocket-Protocol` ?

Reply:
I will extend the auth layer specifically for Cloudlogs to support old clients as well, until we are OK with removing support for them (whenever that happens).

Reply:
Shall we then drop the other ones if they will no longer be used?

Reply:
We could remove `Sec-WebSocket-Protocol` now, but to make the transition easier it is better if we leave it in. If we send all three headers, then k6 will be compatible with both old and new auth layers.